### ISSUE TITLE
Core dump under Wayland

### ISSUE THREAD

**Issue by juxuanu** [josh-richardson/cadmus#51]
Trying to start Cadmus under Wayland results in this:
```
This application failed to start because it could not find or load the Qt platform plugin "wayland"
in "".

Available platform plugins are: eglfs, linuxfb, minimal, minimalegl, offscreen, vnc, xcb.

Reinstalling the application may fix this problem.
[1]    19775 abort (core dumped)  cadmus
```
Other Qt apps work. I have the environment variable `QT_QPA_PLATFORM=wayland` set.

**Comment by josh-richardson** [comment]
Out of interest, have you tried running from source? This may be due to required libraries for Wayland not being included in the app image - at least that's my best guess.

**Comment by juxuanu** [comment]
No, I have not. Sorry, I don't have the computer with Wayland available to me anymore. I can't test further. 
I installed it using the Arch's User Repo (AUR)'s package `cadmus-deb` that uses, logically, the .deb

**Comment by andrewnicolalde** [comment]
Sway user here, setting `QT_QPA_PLATFORM=""` fixed this for me!

This is the case both in the binary version as well as in the AppImage

**Comment by josh-richardson** [comment]
> Sway user here, setting `QT_QPA_PLATFORM=""` fixed this for me!

Good to know. If others can reproduce this, we may have found a fix.

**Comment by lachlanshoesmith** [comment]
> Sway user here, setting `QT_QPA_PLATFORM=""` fixed this for me!
> 
> This is the case both in the binary version as well as in the AppImage

this also worked on gnome 40 for me :)

**Comment by juxuanu** [comment]
It starts under xwayland then...
